FluentTerminal.Models/Resources.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Windows.ApplicationModel.Resources;
+
+namespace FluentTerminal.Models
+{
+    public static class Resources
+    {
+        public static string GetString(string resourceId) =>

```

Comment:
is this only used in `SshConnectionInfoValidationResultExtensions` ? Why not use the already existing `I18N.Translate()`?

Reply:
Because I've needed translations in `FluentTerminal.App.Models` project (i have a method which returns error message there), and `I18N` class isn't accessible there (dependence tree). I was thinking about moving it, but since it is only about a single line of code I've simply written it again, to minimize change.

Reply:
Ok, I see. I think I will move this later on